deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml

```diff
@@ -38,9 +38,6 @@ spec:
           value: "${skaha.hostname}"
         - name: skaha_username
           value: "${skaha.userid}"
-        securityContext:
-          runAsUser: ${skaha.posixid} 
-          runAsGroup: ${skaha.posixid}
```

Comment:
The deletion of these three lines may have got mixed up with the other PR?

Reply:
Yes, since I did a merge after you merged issue #477 changes to master, the code reflects the changes for issue #477.